src/Admin/Settings/API.php

```diff
@@ -29,16 +29,6 @@ class API {
 	 */
 	public $fields = [];
 
-	/**
-	 * Constructor.
-	 *
-	 * @since  1.3.0
-	 * @access public
-	 *
-	 * @return void
-	 */
-	public function __construct() {}
-
```

Comment:
No need for an empty constructor, as otherwise extensions will ask for the `parent::__construct()` and there's no use.